include/COLLADA2GLTFWriter.h

```diff
@@ -35,14 +40,21 @@ namespace COLLADA2GLTF {
 		std::map<COLLADAFW::UniqueId, GLTF::Mesh*> _skinnedMeshes;
 		std::map<COLLADAFW::UniqueId, GLTF::Image*> _images;
 		std::map<COLLADAFW::UniqueId, std::tuple<std::vector<float>, std::vector<float>>> _animationData;
+		bool _use_uint_indices = false;
 
 		bool writeNodeToGroup(std::vector<GLTF::Node*>* group, const COLLADAFW::Node* node);
 		bool writeNodesToGroup(std::vector<GLTF::Node*>* group, const COLLADAFW::NodePointerArray& nodes);
 		GLTF::Texture* fromColladaTexture(const COLLADAFW::EffectCommon* effectCommon, COLLADAFW::SamplerID samplerId);
 		GLTF::Texture* fromColladaTexture(const COLLADAFW::EffectCommon* effectCommon, COLLADAFW::Texture texture);
 
+		void interpolateTranslation(float* base, std::vector<float> input, std::vector<float> output, signed_index_t index,
```

Comment:
What was the motivation behind making these into member functions? I typically leave functions out of the class declaration unless they:
  * need access to class members
  * are published as an interface to be used elsewhere